aiohttp_security/api.py

```diff
@@ -1,12 +1,53 @@
 import asyncio
+import functools
+
 from aiohttp import web
+
 from aiohttp_security.abc import (AbstractIdentityPolicy,
                                   AbstractAuthorizationPolicy)
 
 IDENTITY_KEY = 'aiohttp_security_identity_policy'
 AUTZ_KEY = 'aiohttp_security_autz_policy'
 
 
+def authorize(required=True, redirect_url=None, permission=None):
```

Comment:
Maybe redirection url should be a parameter of `setup()` function, not a property of concrete web-handler?
From my experience all sites has common *login* page which not depends from URL